go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/huandu/xstrings v1.3.2 // indirect
 	github.com/kong/deck v1.7.0
 	github.com/kong/go-kong v0.20.0
-	github.com/kong/kubernetes-testing-framework v0.6.1
+	github.com/kong/kubernetes-testing-framework v0.6.2-0.20210818223452-6200f4fc4f2d
```

Comment:
just a reminder that we need a real release before merging, re: https://github.com/Kong/kubernetes-testing-framework/pull/99